### PR TITLE
Update RegExp named groups identifiers coverage

### PR DESCRIPTION
**Pull request by shvaikalesh**
Ref. https://github.com/tc39/ecma262/pull/1869.
Exported from [r260033](https://trac.webkit.org/changeset/260033/webkit).
All credit goes to @msaboff.

_cc_ @mathiasbynens

### CODE REVIEW

File: test/built-ins/RegExp/named-groups/non-unicode-property-names-invalid.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 the Apple Inc. All rights reserved.
```
Comment (mathiasbynens) [inline test/built-ins/RegExp/named-groups/non-unicode-property-names-invalid.js:1]: "the Apple Inc." sounds a little weird; is this the correct license header?

**Comment by mathiasbynens** [comment]
@bakkot fyi

File: test/built-ins/RegExp/named-groups/non-unicode-property-names-invalid.js (hunk 1 above)
Comment (shvaikalesh) [reply test/built-ins/RegExp/named-groups/non-unicode-property-names-invalid.js:1]: I've borrowed it from other test files authored by JSC team:
* [`/test/built-ins/Object/prototype/toString/proxy-function.js`](https://github.com/tc39/test262/blob/master/test/built-ins/Object/prototype/toString/proxy-function.js)
* [`/test/built-ins/Object/prototype/toString/symbol-tag-non-str-proxy-function.js`](https://github.com/tc39/test262/blob/master/test/built-ins/Object/prototype/toString/symbol-tag-non-str-proxy-function.js)

Comment (mathiasbynens) [reply test/built-ins/RegExp/named-groups/non-unicode-property-names-invalid.js:1]: Given that those are the only two hits for "the Apple Inc." and other files use "Apple Inc.", I'm assuming these are mistakes.

Comment (msaboff) [reply test/built-ins/RegExp/named-groups/non-unicode-property-names-invalid.js:1]: Eliminate "the" and just use "Apple Inc.".

Comment (mathiasbynens) [reply test/built-ins/RegExp/named-groups/non-unicode-property-names-invalid.js:1]: https://www.youtube.com/watch?v=PEgk2v6KntY

Comment (mathiasbynens) [reply test/built-ins/RegExp/named-groups/non-unicode-property-names-invalid.js:1]: https://www.youtube.com/watch?v=PEgk2v6KntY

**Pull request merged**